section2/exercises/if-statements.js

```diff
@@ -31,19 +31,27 @@ if (weather === 'sunny') {
   console.log('good to go!');
 }
 
-/* 
+/*
 YOU DO:
-Using the dogAge variable defined below, 
-determine if a dog is a puppy (2 or younger), 
+Using the dogAge variable defined below,
+determine if a dog is a puppy (2 or younger),
 an adult, or elderly (10 or older).
 
 Log to the console the appropriate age range
 (puppy, adult, elderly).
 */
 
-var dogAge = 3;
-// Write your conditional here
 
+// Write your conditional here
+var dogAge = 11;
+
+if (dogAge <= 2) {
```

Comment:
Nice solution and great use of spaces, indentation etc to make your code very readable.

One thing to just be aware of - the `else if` on line 52 could technically just be an `else` since anything 10 or greater would not fall into the `if` or `else if` conditions. Some might prefer the `else` to be used in that case with the argument that it's more readable because the human doesn't have to talk that 1.5 seconds to interpret the `>= 10` piece and think through what will/won't meet that; whereas the `else` is the known-by-everyone-catch-all.